Dockerfile

```diff
@@ -5,11 +5,8 @@ ENV XDG_CACHE_HOME=/opt/xdg
 
 RUN apk add curl tar bash --no-cache
 RUN set -ex \
-    && curl -sSL https://get.helm.sh/helm-v3.3.1-linux-amd64.tar.gz | tar xz \
+    && curl -sSL https://get.helm.sh/helm-v3.8.2-linux-amd64.tar.gz | tar xz \
```

Comment:
In helm 3.8, OCI is no longer experimental so the line in entrypoint.sh exporting that envvar can be removed.